### ISSUE TITLE
SyntaxError: The requested module 'yargs/yargs' does not provide an export named 'default'

### ISSUE THREAD

**Issue by afdev82** [IBM/audit-ci#340]
Hello,

running audit-ci v7.0.1 I am getting the following error:

```
$ yarn run audit-ci
yarn run v1.22.22
$ /Users/antonio.facciolo/Projects/adNotam/website/node_modules/.bin/audit-ci
file:///Users/antonio.facciolo/Projects/adNotam/website/node_modules/audit-ci/dist/chunk-5NP4NFBX.js:334
import yargs from "yargs/yargs";
       ^^^^^
SyntaxError: The requested module 'yargs/yargs' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:132:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:214:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at async loadESM (node:internal/process/esm_loader:28:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12)

Node.js v20.11.0
error Command failed with exit code 1.
```

with my simple config it's the same:

```
// audit-ci.jsonc
{
  "low": true,
  "show-found": false,
  "skip-dev": true,
  "allowlist": []
}
```

With audit-ci v6 everything worked fine.
How could I solve the issue?
Thank you in advance for your support.

**Comment by quinnturner** [comment]
I have had a ton of issues with yargs, so I am converting our CLI to use `commander`. Will create a PR when ready.

**Comment by quinnturner** [comment]
I think I've fixed this with this PR #341, which I just released in `v7.1.0`. Please test! I hope this isn't breaking other environments... I tried ESM and CommonJS for several Node versions with this new approach. 🤞🏻

**Comment by afdev82** [comment]
Thank you!
Version 7.1.0 is fine.
